main/src/main/scala/sbt/Build.scala

```diff
@@ -8,6 +8,7 @@ import Keys.{ name, organization, thisProject, autoGeneratedProject }
 import Def.{ ScopedKey, Setting }
 
 // name is more like BuildDefinition, but that is too long
+@deprecated("Use .sbt format instead", "0.13.12")
```

Comment:
Right, yeah. There and in the notes as well.

I'll try and sketch one out. I'll need a little bit of your time, as your documentation-fu is stronger than mine.